bundles/org.openhab.binding.venstarthermostat/src/main/java/org/openhab/binding/venstarthermostat/internal/model/VenstarInfoData.java

```diff
@@ -16,25 +16,28 @@
  * The {@link VenstarInfoData} represents a thermostat state from the REST API.
  *
  * @author William Welliver - Initial contribution
+ * @author Matthew Davies - added VenstarAwayMode to include away mode in binding
  */
 public class VenstarInfoData {
     double cooltemp;
     double heattemp;
 
     VenstarSystemState state;
     VenstarSystemMode mode;
+    VenstarAwayMode away;
     int tempunits;
 
     public VenstarInfoData() {
         super();
     }
 
-    public VenstarInfoData(double cooltemp, double heattemp, VenstarSystemState state, VenstarSystemMode mode) {
+    public VenstarInfoData(double cooltemp, double heattemp, VenstarSystemState state, VenstarSystemMode mode,VenstarAwayMode away) {
```

Comment:
```suggestion
    public VenstarInfoData(double cooltemp, double heattemp, VenstarSystemState state, VenstarSystemMode mode, VenstarAwayMode away) {
```

Reply:
You might want to run `mvn spotless:check` in the venstartthermostst directory, it will catch formatting issues like this for you .

Reply:
Done. Thanks for the advice.